daemon/daemon_main.go

```diff
@@ -715,6 +718,10 @@ func initEnv(cmd *cobra.Command) {
 		loadinfo.StartBackgroundLogger()
 	}
 
+	if !option.Config.InstallIptRules {
+		option.Config.Masquerade = false
```

Comment:
I think this is redundant, as `iptables.InstallRules()` installs the MASQ rules and it is not called in the case of `option.Config.InstallIptRules = false`.

Reply:
Yes correct, though wanted to make sure here that noone else is making an assumption about only option.Config.Masquerade in future, hence implicit update.